k8s/namespaces/ccd/ccd-user-profile-api/prod.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: ccd-user-profile-api
+spec:
+  releaseName: ccd-user-profile-api
+  values:
+    java:
+      replicas: 4
+      environment:
+        USER_PROFILE_DB_HOST: ccd-user-profile-api-postgres-db-v11-prod.postgres.database.azure.com
```

Comment:
these vars all need cleaning up, looks like it was forgotten after v11 db work?

Reply:
yes loads of dupes tim, we have ticket to tidy-up